<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesFrame</class>
 <widget class="QFrame" name="PropertiesFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete device</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="deviceTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Control Panel</string>
    </attribute>
    <widget class="QLabel" name="imagelabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>16</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="turnOnOffButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Turn On/Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyValueButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply Value</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Set Date and Time</string>
    </attribute>
    <widget class="QPushButton" name="dateTimeButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>281</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss </string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalValueBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
